dask/array/tests/test_array_function.py

```diff
@@ -231,3 +231,9 @@ def test_like_raises(func):
             RuntimeError, match="The use of ``like`` required NumPy >= 1.20"
         ):
             func(1, like=func((1)))
+
+
+@pytest.mark.skipif(not _numpy_120, reason="NEP-35 is not available")
+@pytest.mark.parametrize("func", [np.array, np.asarray, np.asanyarray])
+def test_like_with_numpy_func(func):
+    assert_eq(func(1, like=da.array((1))), func(1))
```

Comment:
```suggestion
    assert_eq(func(1, like=da.array((1))), func(1))
    
    
@pytest.mark.skipif(not _numpy_120, reason="NEP-35 is not available")
@pytest.mark.parametrize("func", [np.array, np.asarray, np.asanyarray])
def test_like_with_numpy_func_and_dtype(func):
    assert_eq(func(1, dtype=float, like=da.array((1))), func(1, dtype=float))
```

Reply:
We also want to test `like=` together with `dtype=` here. 🙂

Reply:
Sheesh! Thanks :)